src/analyses/variable-sensitivity/abstract_pointer_object.cpp

```diff
@@ -2,23 +2,20 @@
 
  Module: analyses variable-sensitivity
 
- Author: Thomas Kiley, thomas.kiley@diffblue.com
+ Author: Jez Higgins, jez@jezuk.co.uk
```

Comment:
Although the author fields are a bit meaningless in the age of `git praise` / `git blame`, we do prefer to have the name of the original author.

Reply:
I have a feeling this might be something of a git artefact, but happy to make the changes.

Reply:
If these are new files (that git has just decided are moved) then they should have you as the author.  If they are moved they should keep the original author.

Reply:
Honestly, I can't remember, and as you say not hugely important. However, since it obviously looked odd in the diff, I've made the changes.